_config.yml

```diff
@@ -1,9 +1,9 @@
 # Site settings
 permalink: /blog/:year/:month/:day/:title.html
-title: ProjectName
-tagline: This is a brief one-liner about the project
+title: MyRocks
+tagline: A RocksDB storage engine with MySQL
 description: >
-  This is a longer description of the project, a bit of a sales-pitch with a bit of technical information included.
+  Benefit from all the features of MySQL while using RocksDB as backend storage
 fbappid: "1615782811974223"
 gacode: "UA-12345-6"
```

Comment:
We can do the GA id together. I will need to add something to our internal GA page, I think.

I believe the fbappid stays the same across all things using this template.